mkosi/qemu.py

```diff
@@ -191,8 +194,20 @@ async def notify() -> None:
         logging.debug(f"Received {num_messages} notify messages totalling {format_bytes(num_bytes)} bytes")
 
 
+@contextlib.contextmanager
+def copy_ephemeral(config: MkosiConfig, src: Path) -> Iterator[Path]:
+    src = src.resolve()
+    tmp = Path(src.parent / f"{src.name}-{uuid.uuid4().hex}")
```

Comment:
This doesn't use `tempfile` because you only need a temporary name instead of a directory for the btrfs snapshot? If so, a comment would probably make sense.